lib/PuppeteerSharp/WaitTaskTimeoutException.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace PuppeteerSharp
+{
+    [Serializable]
+    internal class WaitTaskTimeoutException : Exception
```

Comment:
base class should be `PuppeteerException`

Reply:
@Meir017 genius. Done.